felucca/backend/common/status.py

```diff
@@ -0,0 +1,3 @@
+from enum import Enum
+
+Status = Enum("Status", ("Undefined", "Pending", "Running", "Success", "Error", "Failed", "Killed"))
```

Comment:
when I thinking of an enum class: I am thinking of something like this:
>>> from enum import Enum
>>> class Color(Enum):
...     RED = 1
...     GREEN = 2
...     BLUE = 3
See (https://docs.python.org/3/library/enum.html)
In your implementation, can you map the string to the integer which you will store in the database?